astrokat/noisediode.py

```diff
@@ -0,0 +1,67 @@
+import time
+
+from simulate import user_logger
+
+
+# switch noise-source pattern off
+def on(mkat, logging=True):
+    if logging:
+        user_logger.info('Switch noise-diode on')
+    # Noise Diodes are triggered on all antennas in array simultaneously
+    # add a second to ensure all digitisers set at the same time
+    timestamp = time.time() + 1
```

Comment:
Rather change this `1` second to a constant at the start of the module - it is used in a few places.  E.g.
```
NOISE_DIODE_REQUEST_LEAD_TIME = 1
```